UI/window-projector.cpp

```diff
@@ -4,6 +4,7 @@
 #include <QMenu>
 #include <QScreen>
 #include <qt-wrappers.hpp>
+#include "moc_window-projector.cpp"
```

Comment:
Interesting that this is not paired with a pre-existing line.

Reply:
That's because `window-basic-main.hpp` includes `window-projector.hpp` - another case of compilation being possible out of sheer luck.